platformio.ini

```diff
@@ -157,6 +162,37 @@ lib_deps =  ${env.lib_deps} ${env.libs_gps} ${env.libs_ble}
 board_build.partitions = no_ota.csv
 
 
+; ================== T-BEAM ================== 
+[env:ttgo_lora_v12]
```

Comment:
to stay in line with the other profiles the base name should be without display drivers and builds containing the display driver should have a `_display_` in their name to make it clear

Reply:
I've appended `_display` to the names in 97fdc0c , as requested. 

However, introducing an `[env:*]` for every optional feature seems a quiet large overhead, thus I did not add non-display options for this board.

This discussion seems to be a bit out of scope in this PR, thus let's discuss it in #14.